deepchecks/vision/base.py

```diff
@@ -409,6 +396,13 @@ def run(
                 run_train_test_checks=run_train_test_checks,
                 results=results
             )
+            for check_idx, check in checks.items():
+                if check_idx not in results:
+                    if str(check_idx).endswith('Train'):
```

Comment:
IMO to compare strings is not ideal, I'd do 3 dicts (checks, single_test, single_train)